rnaseq-cgl-pipeline/wrapper.py

```diff
@@ -0,0 +1,96 @@
+import argparse
+import os
+import shutil
+import subprocess
+import json
+import logging
+from uuid import uuid4
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger()
+
+
+def call_pipeline(mount, args):
+    uuid = 'Toil-RNAseq-' + str(uuid4())
+    if not os.path.exists(mount + uuid):
+        os.makedirs(os.path.join(mount, uuid))
+    os.environ['PYTHONPATH'] = '/opt/rnaseq-pipeline/src'
+    command = ['python', '-m', 'toil_scripts.rnaseq_cgl.rnaseq_cgl_pipeline',
+               os.path.join(mount, 'jobStore'),
+               '--retryCount', '1',
+               '--output-dir', mount,
+               '--workDir', os.path.join(mount, uuid),
+               '--star-index', 'file://'+ args.star,
+               '--rsem-ref', 'file://' + args.rsem,
+               '--kallisto-index', 'file://' + args.kallisto,
+               '--sample-urls']
+    command.extend(['file://' + x for x in args.samples])
+    if args.restart:
+        command.append('--restart')
+    try:
+        subprocess.check_call(command)
+    finally:
+        user_id = subprocess.check_output(['stat', '-c', '%u:%g', mount]).strip()
+        subprocess.check_call(['chown', '-R', user_id, mount])
```

Comment:
I'd propose os.chown() but it's not recursive and shutil.chown() exists only in Python 3.